crates/ra_fmt2/src/dsl.rs

```diff
@@ -0,0 +1,198 @@
+use crate::pattern::{Pattern, PatternSet};
+use crate::trav_util::{next_non_whitespace_sibling, prev_non_whitespace_sibling};
+use ra_syntax::{SyntaxElement, SyntaxKind::*};
+
+/// `SpacingRule` describes whitespace requirements between `SyntaxElement` Note
```

Comment:
```suggestion
/// `SpacingRule` describes whitespace requirements between `SyntaxElement`.
/// Note that it doesn't handle indentation (first whitespace on a line), there's
```

Putting the `Note` on the next line will prevent it from appearing in the documentation summary when showing the `dsl` module in rustdoc.